rclcpp/include/rclcpp/publisher.hpp

```diff
@@ -116,6 +121,26 @@ class PublisherBase
   const rcl_publisher_t *
   get_publisher_handle() const;
 
+  // RCLCPP_PUBLIC
+  template<typename EventCallbackT>
+  void
+  add_event_handle(
+    const EventCallbackT & callback,
+    const rcl_publisher_options_t & publisher_options,
```

Comment:
Publisher option is passed in as part of the constructor, but it is never stored as part of the class. Also, this `add_event_handle()` function is called during the publisher's construction.

Reply:
If it's just used as a helper function for the construction then let's make it a class private function instead of public and get rid of the comment `// RCLCPP_PUBLIC`

Reply:
I will make it a protected function.